app/webhook.go

```diff
@@ -95,21 +95,22 @@ func (a *App) handleWebhookEvents(c request.CTX, post *model.Post, team *model.T
 }
 
 func (a *App) TriggerWebhook(c request.CTX, payload *model.OutgoingWebhookPayload, hook *model.OutgoingWebhook, post *model.Post, channel *model.Channel) {
```

Comment:
It looks like the unit tests in https://github.com/mattermost/mattermost-server/blob/master/app/webhook_test.go do not test the case where there are multiple callback URLs. @daran9 Are you open to adding a test there that takes multiple URLs into account?

Reply:
Yes definitely. I was wondering how I should test this!

Reply:
@daran9 there is a unit test here with a single callback URL, which you can use as a reference:
https://github.com/mattermost/mattermost-server/blob/master/app/webhook_test.go#L590

You can add another test which tests with multiple callback URLs.

Reply:
I have added the requested tests; Please take a look and let me know. Thanks!